MOBT-3-Patterns/MVVMWithoutC/SignOut/SignOutViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MainPageViewController.swift
+//  MVVMWithoutC
+//
+//  Created by Mykyta Babanin on 01.07.2021.
+//
+
+import UIKit
+
+fileprivate enum Consts {
+    static let signOutTitle = "Sign Out"
+}
+
+class SignOutViewController: UIViewController {
+    
+    var viewModel: ViewModelProtocol!
```

Comment:
Looks like the approach to usage of ViewModel is not consistent within the project, in one place you mark it as optional and treat it as one, but here you use it as an implicitly unwrapped optional.
Let's make sure we use the same approach in all places of our codebase.

My suggestion would be to use the [instantiateViewController(identifier:creator:)](https://developer.apple.com/documentation/uikit/uistoryboard/3213989-instantiateviewcontroller) initializer (please be aware it's only available in iOS 13.x and newer) and make the viewModel non-optional. You can find a short tutorial on how to use this initializer [here](https://www.hackingwithswift.com/example-code/uikit/how-to-use-dependency-injection-with-storyboards).